modules/light-clients/09-localhost/client_state.go

```diff
@@ -57,11 +61,7 @@ func (cs ClientState) Initialize(ctx sdk.Context, cdc codec.BinaryCodec, clientS
 		return sdkerrors.Wrap(clienttypes.ErrInvalidConsensus, "initial consensus state for localhost must be nil.")
 	}
 
-	clientState := ClientState{
```

Comment:
Why was this removed?

Reply:
Because `Initialize` is an interface function, and previously we were not actually using the method receiver. The usage looks like

```go
func (k Keeper) CreateLocalhostClient(ctx sdk.Context) error {
	clientState := localhost.NewClientState(types.GetSelfHeight(ctx), k.GetParams(ctx).IsAllowedClient(exported.Localhost))
	return clientState.Initialize(ctx, k.cdc, k.ClientStore(ctx, exported.Localhost), nil)
}
```

This enables us to configure the clientState.Enabled field based on the `allowed_clients` list.